digitalocean/droplet/resource_droplet.go

```diff
@@ -580,6 +652,27 @@ func resourceDigitalOceanDropletUpdate(ctx context.Context, d *schema.ResourceDa
 		}
 	}
 
+	if d.HasChange("backup_policy") {
+		var newPolicy godo.DropletBackupPolicyRequest
+		bp, ok := d.GetOk("backup_policy")
+		if ok && bp != nil { // backup_policy must be set.
+			_, new := d.GetChange("backup_policy")
+			if newList, ok := new.([]interface{}); ok && len(newList) != 0 {
+				newPolicy = new.([]interface{})[0].(godo.DropletBackupPolicyRequest)
```

Comment:
If I attempt to add a backup policy to an existing Droplet, it panics here with:

```
panic: interface conversion: interface {} is map[string]interface {}, not godo.DropletBackupPolicyRequest
```

This needs to be handled similarly to how you did it in `resourceDigitalOceanDropletCreate`. I'd recommend breaking out that logic into a new function named something like `expandBackupPolicy`.